client/modules/IDE/components/Toast.jsx

```diff
@@ -2,15 +2,17 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
+import { useTranslation } from 'react-i18next';
 import * as ToastActions from '../actions/toast';
 
 import ExitIcon from '../../../images/exit.svg';
 
 function Toast(props) {
+  const { t } = useTranslation('common');
   return (
     <section className="toast">
       <p>
-        {props.text}
+        {t(props.text)}
```

Comment:
After talking to @oruburos, `props.text` will be passed through and displayed. This is fine in the short-term since the Editor UI will continue to work whilst we translate the ap in follow-up PRs.